src/Cms/Language.php

```diff
@@ -454,6 +495,16 @@ protected function setUrl(string $url = null): self
         return $this;
     }
 
+    /**
+     * Returns the slug rules for this language
```

Comment:
I'd call it "custom slug rule overrides" as the method only returns the data that is manually configured, not the rules themselves.